services/ui-src/src/components/app/AppRoutes.tsx

```diff
@@ -21,8 +21,11 @@ import { ScrollToTopComponent, useUser } from "utils";
 import { Fragment } from "react";
 
 export const AppRoutes = () => {
-  const { userIsAdmin } = useUser().user ?? {};
+  const { userIsAdmin, userReports } = useUser().user ?? {};
   const mlrReport = useFlags()?.mlrReport;
+  // determine if the user has access to specific reports
+  const hasMcpar = userReports?.includes("MCPAR") || userIsAdmin;
+  const hasMlr = userReports?.includes("MLR") || userIsAdmin;
```

Comment:
Nit: Would maybe name these `hasMcparAccess` or similar. Alternatively could make a lookup object that has something like: 
```
userReportAccess = {
  MCPAR: userReports?.includes("MCPAR") || userIsAdmin,
  MLR: userReports?.includes("MLR") || userIsAdmin,
}
```
then access like:
```
userReportAccess["MCPAR"] ? whatever : whatever
```